### PR TITLE
[FIX] product_configurator_mrp (Variant BoM Components Duplicated)

### PR DESCRIPTION
**Pull request by patrickrwilson**
Fix for issue https://github.com/OCA/product-configurator/issues/59

When there are multiple components that use the same configuration set on the parent bom, duplicate components would be generated on the variant bom.

### CODE REVIEW

**Comment by patrickrwilson** [comment]
ping @dreispt

**Review by dreispt — APPROVED** [review]

**Comment by dreispt** [comment]
@patrickrwilson I resolved merge conflicts, but made pre-commit unhappy. Can you check please?

File: product_configurator_mrp/models/product_config.py [hunk 1]
```diff
@@ -69,28 +69,25 @@ def create_get_bom(self, variant, product_tmpl_id=None, values=None):
...
-                                    values = updates.get("value", {})
-                                    values = self.get_vals_to_write(
-                                        values=values, model="mrp.bom.line"
-                                    )
-                                    values.update(parent_bom_line_vals)
-                                    bom_lines.append((0, 0, values))
+                            if parent_bom_line.bom_id.id == parent_bom.id:
+                                parent_bom_line_vals = {
+                                    "product_id": parent_bom_line.product_id.id,
+                                    "product_qty": config_bom_line.product_qty,
```
Comment (dreispt) [inline product_configurator_mrp/models/product_config.py]: I realize you used a personal fork for this PR, so suggesting the fix here:
```suggestion
                                    "product_qty": parent_bom_line.product_qty,
```

**Review by dreispt — APPROVED** [review]

**Pull request merged**